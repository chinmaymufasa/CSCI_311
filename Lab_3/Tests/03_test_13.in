2
45
5
2
77
5
2
129
5
2
40
5
2
171
5
2
54
5
2
137
5
2
176
5
2
197
5
2
73
5
2
175
5
2
23
5
2
119
5
2
137
5
2
24
5
2
177
5
2
161
5
2
156
5
2
128
5
2
131
5
-1
