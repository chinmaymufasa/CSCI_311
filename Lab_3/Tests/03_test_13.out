45 
45 77 
45 77 129 
40 45 77 129 
40 45 77 129 171 
40 45 54 77 129 171 
40 45 54 77 129 137 171 
40 45 54 77 129 137 171 176 
40 45 54 77 129 137 171 176 197 
40 45 54 73 77 129 137 171 176 197 
40 45 54 73 77 129 137 171 175 176 197 
23 40 45 54 73 77 129 137 171 175 176 197 
23 40 45 54 73 77 119 129 137 171 175 176 197 
23 40 45 54 73 77 119 129 137 137 171 175 176 197 
23 24 40 45 54 73 77 119 129 137 137 171 175 176 197 
23 24 40 45 54 73 77 119 129 137 137 171 175 176 177 197 
23 24 40 45 54 73 77 119 129 137 137 161 171 175 176 177 197 
23 24 40 45 54 73 77 119 129 137 137 156 161 171 175 176 177 197 
23 24 40 45 54 73 77 119 128 129 137 137 156 161 171 175 176 177 197 
23 24 40 45 54 73 77 119 128 129 131 137 137 156 161 171 175 176 177 197 
